app/src/main/java/com/example/lunabeeusers/data/model/User.kt

```diff
@@ -12,4 +18,22 @@ data class User(
     @Json(name = "email") val email: String,
     @Json(name = "gender") val gender: String,
     @Json(name = "avatar") val imgSrcUrl: String
-) : Parcelable
\ No newline at end of file
+) : Parcelable, AbstractBindingItem<ItemUserListBinding>() {
+
+    override val type: Int
+        get() = R.id.fastadapter_item
+
+    override fun bindView(binding: ItemUserListBinding, payloads: List<Any>) {
+        binding.firstnameTv.text = firstname
+        binding.lastnameTv.text = lastname
+        loadImageFromUrl(binding.userAvatarIv, imgSrcUrl)
+    }
+
+    override fun createBinding(inflater: LayoutInflater, parent: ViewGroup?): ItemUserListBinding {
+        return ItemUserListBinding.inflate(inflater, parent, false)
+    }
```

Comment:
It would be better if the UI part could be outside of the data part

Reply:
fix in Sprint3